org-code-javabuilder/lang/README.md

```diff
@@ -0,0 +1,13 @@
+# Lang
```

Comment:
🎉 Thanks for adding a readme!!! Mind adding a note in the org-code-javabuilder readme too?  (can be done as a follow-up) https://github.com/code-dot-org/javabuilder/blob/main/org-code-javabuilder/README.md

Reply:
done, thanks for the reminder!